### PR TITLE
JNG-4531 Release karaf-jasypt-support

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4531" title="JNG-4531" target="_blank">Sub-taskJNG-4531</a>  Release karaf-jasypt-support
  </td></table>
  <br />

### CODE REVIEW

File: .flattened-pom.xml [hunk 1]
```diff
@@ -8,15 +8,31 @@
   <packaging>bundle</packaging>
```
Comment (robertcsakany) [inline .flattened-pom.xml]: .flattened have to be ignored

**Review by robertcsakany — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -22,5 +22,4 @@ nb*.xml
 **/*.iml
 
 
-
```
Comment (bencegelei) [inline .gitignore:25]: Unnecessary change.
Since `.flattened-pom.xml` is already tracked only (re-)adding to gitignore won't work. It must be explicitly deleted.

For reference see:
- https://git-scm.com/docs/gitignore#_description
- https://git-scm.com/docs/gitignore#_notes
- https://git-scm.com/docs/git-rm#Documentation/git-rm.txt---cached

**Review by bencegelei — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -22,5 +22,4 @@ nb*.xml
...
-**/.flattened-pom.xml
```
Comment (bencegelei) [inline .gitignore:26]: Unnecessary change.
Since `.flattened-pom.xml` is already tracked only (re-)adding to gitignore won't work. It must be explicitly deleted.

For reference see:
- https://git-scm.com/docs/gitignore#_description
- https://git-scm.com/docs/gitignore#_notes
- https://git-scm.com/docs/git-rm#Documentation/git-rm.txt---cached

**Review by bencegelei — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (gaborflorian) [reply .gitignore:26]: [90fb16d](https://github.com/BlackBeltTechnology/karaf-jasypt-support/pull/2/commits/90fb16dae3be9705ff56e77a2b4bc60309c32233)

File: .flattened-pom.xml (hunk 1 above)
Comment (gaborflorian) [reply .flattened-pom.xml]: [90fb16d](https://github.com/BlackBeltTechnology/karaf-jasypt-support/pull/2/commits/90fb16dae3be9705ff56e77a2b4bc60309c32233)

**Review by bencegelei — APPROVED** [review]

File: pom.xml [hunk 4]
```diff
@@ -242,6 +293,99 @@
...
+                <executions>
+                    <execution>
+                        <id>attach-javadocs</id>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
```
Comment (noherczeg) [inline pom.xml:397]: Missing license genration

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 4 above)
Comment (gaborflorian) [reply pom.xml:397]: [647ba71](https://github.com/BlackBeltTechnology/karaf-jasypt-support/pull/2/commits/647ba716b6847e0b04c7b532ed502285228d0463)

**Review by robertcsakany — APPROVED** [review]

**Review by noherczeg — APPROVED** [review]
